Src/IronPython/Runtime/MemoryView.cs

```diff
@@ -385,15 +395,53 @@ public sealed class MemoryView : ICodeFormattable, IWeakReferenceable {
             int firstByteIndex = _start + index * _itemsize * _step;
             object result = packBytes(format, getByteRange(firstByteIndex, _itemsize), 0, (int)_buffer.ItemSize);
 
-            // TODO: BigInteger should also be merged into an integer once the int/long merge occurs
-            if (result is BigInteger || result is double || result is float || result is Bytes) {
+            // TODO: BigInteger/long/ulong should also be merged into an integer once the int/long merge occurs
+           if (result is BigInteger || result is double || result is float || result is Bytes) {
```

Comment:
`float` probably needs to be cast to `double`.

I wonder if there should be a .NET primitive to Python type helper somewhere? Maybe there already is...

Reply:
I believe `GetPythonType` works for primitive types. I'm not sure if that's the functionality you're referring to.

Reply:
I meant something that essentially does what you do here: `byte`/`short`->`int`, `float`->`double`, `long`->`BigInteger`, and so on for all C# primitive types. `GetPythonType` does not do this since it will return you the Python version of the C# types (e.g. it will give you `System.Int16` for shorts).

Reply:
I wasn't able to find anything that does this, so I added one such method in `PythonOps`. I'm not entirely sure what the correct location for this would be.